tldr.py

```diff
@@ -507,8 +508,8 @@ def main() -> None:
         print(get_commands(options.platform))
     elif options.render:
         for command in options.command:
-            if os.path.exists(command):
-                with open(command, encoding='utf-8') as open_file:
+            if Path(command).exists:
```

Comment:
This is a method call
```suggestion
            if Path(command).exists():
```